.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.vscode/settings.json
```

Comment:
You could add your dist folder to this so it doesn't try to upload your build files

Reply:
should, not could